roles/installer/ospd/overcloud/scale/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+- name: Creating scale script.
+  shell: "cp -pv ~/overcloud_deploy.sh ~/overcloud_scale.sh"
+
+- name: change amount of scaled nodes
+  replace:
+      dest: "~/overcloud_scale.sh"
+      regexp: '^(\s*)--{{ scale_service }}-scale\s\d\s\\'
```

Comment:
Is it mandatory to have the beginning of the line regex? '^(\s*)
why not "--{{ scale_service }}-scale.*$" ?